src/DistributedLock.Tests/Tests/Redis/RedisDistributedLockTest.cs

```diff
@@ -2,13 +2,14 @@
 using Moq;
 using NUnit.Framework;
 using StackExchange.Redis;
+using System.Globalization;
 
 namespace Medallion.Threading.Tests.Redis;
 
-[Category("CI")]
+[NonParallelizable] // one of the tests temporarily changes CurrentCulture
```

Comment:
We shouldn't need this. CurrentCulture is an asynclocal value (this is critical since otherwise you couldn't set different cultures for different concurrent web requests), so it shouldn't interfere with any concurrently-executing tests.

We only need to worry about this when setting truly-global values like environment variables that affect all threads at once.